src/test/java/teammates/test/cases/action/InstructorFeedbackEditSaveActionTest.java

```diff
@@ -139,14 +139,14 @@ public void testExecuteAndPostProcess() {
                 + "<Text: instructions>|||/page/instructorFeedbackEditSave";
         AssertHelper.assertLogMessageEquals(expectedString, a.getLogMessage());
 
-        ______TS("success: Masquerade mode, never release results, invalid timezone and graceperiod");
+        ______TS("success: Masquerade mode, 'custom' release results, invalid timezone and graceperiod");
```

Comment:
Use `manual` instead of `'custom'`?

Reply:
Sorry my bad. The actual test is set as manual only. Will fix it.